scripts/index.js

```diff
@@ -51,4 +50,15 @@ projects.forEach(function(project) {
   $('#portfolio-pages').append(project.toHtml());
 });
 
+Project.fetchAll = function() {
+  $.getJSON('data/project_data.JSON')
```

Comment:
the getJSON method is just getting all the data (in the form of JSON) from whatever directory you specify (in this case it is in the data directory in the project_data.JSON file). It is just a shorthand way of getting data, sometimes I find it easier to write it out longhand, where you specify the method (GET), the type of data you're expecting (JSON), etc. See this documentation for more clarification: http://api.jquery.com/jquery.getjson/ for reference.